powerauth-push-server/pom.xml

```diff
@@ -123,7 +123,20 @@
 			<scope>test</scope>
 		</dependency>
 
-	</dependencies>
+		<!--swagger-->
```

Comment:
Please check the formatting, it seems to be inconsistent.

Reply:
I will try to fix it.